.pre-commit-config.yaml

```diff
@@ -0,0 +1,7 @@
+repos:
+- repo: https://github.com/dnephin/pre-commit-golang
+  rev: v0.4.0
+  hooks:
+    - id: go-fmt
+    - id: golangci-lint
```

Comment:
I think there may be some redundancy here, since `go-fmt` is run as a part of `golangci-lint`. We should make sure our current developer guides reflect this.